shapely/constructive.py

```diff
@@ -887,10 +887,23 @@ def simplify(geometry, tolerance, preserve_topology=True, **kwargs):
 def snap(geometry, reference, tolerance, **kwargs):
     """Snaps an input geometry to reference geometry's vertices.
 
-    The tolerance is used to control where snapping is performed.
+    Vertices of the first geometry are snapped to vertices of the second.
+    geometry. The input geometries are not modified.
     The result geometry is the input geometry with the vertices snapped.
     If no snapping occurs then the input geometry is returned unchanged.
 
+    The tolerance is used to control where snapping is performed.
+    A snap distance tolerance is used to control where snapping is performed.
```

Comment:
Those two sentences are a bit duplicative?

Reply:
oh right, I'll remove that